titiler/xarray/factory.py

```diff
@@ -440,7 +440,7 @@ def map_viewer(
                 tilejson_url += f"?{urlencode(request.query_params._list)}"
 
             tms = self.supported_tms.get(TileMatrixSetId)
-            return templates.TemplateResponse(
+            return self.templates.TemplateResponse(
```

Comment:
thanks for catching this. I did make a small breaking change in the last titiler release 😓 

I fixed this in https://github.com/developmentseed/titiler-xarray/commit/b9660164c9154ec8f30ef990f76effba3488caa2